src/libraries/System.Net.Quic/tests/FunctionalTests/QuicConnectionTests.cs

```diff
@@ -115,6 +115,11 @@ public async Task Dispose_WithPendingAcceptAndConnect_PendingAndSubsequentThrowO
 
                     // Pending ops should fail
                     await Assert.ThrowsAsync<QuicOperationAbortedException>(() => acceptTask);
+
+                    // TODO: This may not always throw QuicOperationAbortedException due to a data race with MsQuic worker threads
+                    // (CloseAsync may be processed before OpenStreamAsync as it is scheduled to the front of the operation queue)
+                    // To be revisited once we standartize on exceptions.
+                    // [ActiveIssue("https://github.com/dotnet/runtime/issues/55619")]
                     await Assert.ThrowsAsync<QuicOperationAbortedException>(() => connectTask);
```

Comment:
Should there also be `ThrowsAnyAsync` based on the comment you just added?

Reply:
fixed